src/Storages/tests/gtest_transform_query_for_external_database.cpp

```diff
@@ -249,3 +249,22 @@ TEST(TransformQueryForExternalDatabase, Strict)
     /// !isCompatible() takes place
     EXPECT_THROW(check(state, 1, "SELECT column FROM test.table WHERE left(column, 10) = RIGHT(column, 10) AND SUBSTRING(column FROM 1 FOR 2) = 'Hello'", ""), Exception);
 }
+
+TEST(TransformQueryForExternalDatabase, Null)
+{
+    const State & state = State::instance();
+
+    check(state, 1,
+          "SELECT field FROM table WHERE field IS NULL",
+          R"(SELECT "field" FROM "test"."table" WHERE "field" IS NULL)");
+    check(state, 1,
+          "SELECT field FROM table WHERE field IS NOT NULL",
+          R"(SELECT "field" FROM "test"."table" WHERE "field" IS NOT NULL)");
+
+    check(state, 1,
+          "SELECT field FROM table WHERE isNull(field)",
+          R"(SELECT "field" FROM "test"."table" WHERE "field" IS NULL)");
```

Comment:
Where in code `isNull` is transformed into `... IS NULL` ? I don't see it in this PR, was it supported before and we need only allow it?

Reply:
>Where in code isNull is transformed into ... IS NULL ?

Yes, it is done in #29446